app/console

```diff
@@ -13,7 +13,7 @@
 set_time_limit(0);
 
 require_once __DIR__.'/bootstrap.php.cache';
-require_once __DIR__.'/CliKernel.php';
+require_once __DIR__.'/AppKernel.php';
```

Comment:
I think the benefit was minor, it was confusing compared to Symfony standard and generally running commands with a different set of bundles than web instance sounds fragile to me. I was also often getting issue when generating migrations. I think we are fine with these 3 bundles in default kernel. Makes things simpler.